parlai/core/agents.py

```diff
@@ -266,10 +272,11 @@ def create_agent_from_model_file(model_file, opt_overides=None):
         The agent
     """
     opt = {}
-    opt['model_file'] = model_file
-    if opt_overides is None:
-        opt_overides = {}
-    opt['override'] = opt_overides
+    add_datapath_and_model_args(opt)
```

Comment:
so this is really the fundamental change in this pr right?

Reply:
Yep!